### DISCUSSION TITLE
0.5.8

### DISCUSSION THREAD

**Discussion by milesisbetterxoxo** [cheblol/FNF-FunkinGoldyEngine-lua#2] [Show and tell]
sorry out of ideas.
0.5.8
* again, fixed health drain.
* new icon system!!!! (check assets images icons folder for a example)

<hr /><em>This discussion was created from the release <a href='https://github.com/cheblol/FNF-FunkinGoldyEngine-lua/releases/tag/0.5.8'>0.5.8</a>.</em>

**Comment by Meloomazy** [comment]
bro how do i use this function lua?

`addGlitchEffect('cheaterbackground',5,0.5,0.6)`
is that correct?
its work but it wont moving it just stand still with the effect on

**Reply by milesisbetterxoxo** [reply]
> bro how do i use this function lua?
> 
> `addGlitchEffect('cheaterbackground',5,0.5,0.6)` is that correct? its work but it wont moving it just stand still with the effect on

weird, it must work correctly
it worked for me in this version
EDITED : perhaps, i didnt edit shaders

**Comment by Meloomazy** [comment]
![Annotation 2022-06-10 215023](https://user-images.githubusercontent.com/92463009/173092012-97ddf20a-922c-4254-a795-11a7b317fc7f.png)
